deploy/stripe-ct/action.yml

```diff
@@ -0,0 +1,83 @@
+name: Deploy Storefront
+description: Builds Middleware app docker image
```

Comment:
```suggestion
description: Deploy Stripe for Commercetools app
```

Reply:
Or `Deploy Storefront with Stripe for Commercetools` if we gonna add middleware deployment here as well

Reply:
```suggestion
description: Deploy Storefront with Stripe for Commercetools
```